src/features/dotnetTest.ts

```diff
@@ -435,6 +458,46 @@ export default class TestManager extends AbstractProvider {
             listener.dispose();
         }
     }
+
+    private async _getLaunchConfigurationForVSTestInContext(
+        fileName: string,
+        line: number,
+        column: number,
+        runSettings: string,
+        targetFrameworkVersion: string,
+        debugEventListener: DebugEventListener): Promise<LaunchConfiguration | null> {
+
+        const listener = this._server.onTestMessage(e => {
+            this._eventStream.post(new DotNetTestMessage(e.Message));
+        });
+
+        const request: protocol.V2.DebugTestsInContextGetStartInfoRequest = {
+            FileName: fileName,
+            Line: line + 1,
```

Comment:
is this due to vscode using zero-indexed positions while roslyn uses one-indexed positions?

Reply:
No. It's because the deserializer expects the incoming position to be 1-indexed. Both vscode and roslyn actually expect 0-indexed.